CCDB/src/CcdbApi.cxx

```diff
@@ -756,7 +757,12 @@ bool CcdbApi::initTGrid() const
     mAlienInstance = TGrid::Connect("alien");
     static bool errorShown = false;
     if (!mAlienInstance && errorShown == false) {
-      LOG(error) << "TGrid::Connect returned nullptr. May be due to missing alien token";
```

Comment:
In case the alien-token is not present, `TGrid::Connect` will ask for the password before checking if it is allowed to go on w/o the token. I thought we want to avoid this. I think in the long run we should find a way (ask for the extension of the `TGrid::Connect`) to call Connect with a simple check for the existing token, w/o demanding a password.

Reply:
We could also add the check elsewhere (in a script at the start of O2DPG/full-system-test/sim-challenge) and add in documentation that ALIEN GRID tokens are a required.

Reply:
> You are right, we should add such a check without asking for a password. But I think for now this is the best we can do.

I disagree. Passwordless checks are possible and not much harder.